lib/wasp/WASP.cpp

```diff
@@ -455,12 +456,15 @@ vector <size_t> vdiff(vector <size_t> a, vector <size_t> b) {
 	for (int i=0; i<a.size(); i++) c[i] = a[i] - b[i];
 	return(c);
 }
+#endif
 
 // Determine POD type 
 //
+#ifdef VAPOR3_0_0_ALPHA
```

Comment:
If this is commented out a call to NetCDFType((float)3.0f) will be promoted to NetCDFType(double) without warning

Reply:
fixed in commit 4d331b0